tdrs-frontend/src/actions/reports.js

```diff
@@ -174,7 +175,7 @@ export const submit =
       )
     })
 
-    Promise.all(submissionRequests)
+    return Promise.all(submissionRequests)
```

Comment:
This return doesn't really change the behavior, but was needed in order to assert asynchronous actions were recorded in tests.